### PR TITLE
Change requirements to IDF >5.0 (AIV-660)

### PR DESCRIPTION
**Pull request by Ilidur**
Updated documentation to reflect min requirement.
The change has been tested with IDF 5.2.
The following tests have been done:
Tested and ran a build of `examples/human_face_detect` on Seeed Xiao ESP32S3 with correct output. Unsure of the .rst files.
zh_CN translation using online translation tool.

### CODE REVIEW

File: examples/cat_face_detect/main/idf_component.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dependencies:
   # Required IDF version
-  idf: "5.0.*"
+  idf: ">5.0"
```
Comment (BlueSkyB) [inline examples/cat_face_detect/main/idf_component.yml:3]: 5.0 should also be included. You can use `idf: ">=5.0.0"`

File: examples/color_detect/main/idf_component.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 dependencies:
   # Required IDF version
-  idf: "5.0.*"
+  idf: ">5.0"
```
Comment (BlueSkyB) [inline examples/color_detect/main/idf_component.yml:3]: 5.0 should also be included. You can use `idf: ">=5.0.0"`

File: examples/face_recognition/main/idf_component.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 dependencies:
   # Required IDF version
-  idf: "5.0.*"
+  idf: ">5.0"
```
Comment (BlueSkyB) [inline examples/face_recognition/main/idf_component.yml:3]: 5.0 should also be included. You can use `idf: ">=5.0.0"`

File: examples/human_face_detect/main/idf_component.yml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 dependencies:
   # Required IDF version
-  idf: "5.0.*"
+  idf: ">5.0"
```
Comment (BlueSkyB) [inline examples/human_face_detect/main/idf_component.yml:3]: 5.0 should also be included. You can use `idf: ">=5.0.0"`

File: idf_component.yml [hunk 5]
```diff
@@ -1,3 +1,3 @@
 description: esp_dl provides APIs for Neural Network (NN) Inference, Image Processing, Math Operations and some Deep Learning Models.
 dependencies:
-  idf: "5.0.*"
+  idf: ">5.0"
```
Comment (BlueSkyB) [inline idf_component.yml:3]: 5.0 should also be included. You can use `idf: ">=5.0.0"`

**Comment by Ilidur** [comment]
Thank you @BlueSkyB for the review. The requested changes have been added.

**Pull request merged**